### DISCUSSION TITLE
Database file separation

### DISCUSSION THREAD

**Discussion by tushitdave** [explosion/spaCy#7888] [Help: Other Questions]
Using Prodigy tool and after Annotations we have created Database file which can be called to run NER model to check F1score, Precision and Recall. 
Now that we have implemented this successfully, We have two questions:- 

Q.1) - Is there any option we can separate One Database file to 3-4 Database file and check NER model performance. 

Q.2) - Is there any limitations in labelling Tags ? We would like to avoid multiple iterations for single document.

- GitHub Discussions: https://github.com/explosion/spaCy/discussions
- Stack Overflow: http://stackoverflow.com/questions/tagged/spacy
-->

## Your Environment

- Operating System:
- Python Version Used:
- spaCy Version Used:
- Environment Information:

**Comment by polm** [comment]
> Is there any option we can separate One Database file to 3-4 Database file and check NER model performance.

How do you want to split the data? 

Annotations are saved as JSON blobs, so it would typically be easiest to dump the data to JSON, filter it with whatever criteria you have, and then use that as training data. 

> Is there any limitations in labelling Tags ? We would like to avoid multiple iterations for single document.

Not sure I understand the question, can you give an example of what you want to do? One of the main limitations with NER tags is that they can't be overlapping, so any token can only have one tag. Other than that there are things you shouldn't do, or things that cause problems, but not many strict limitations.
